js/pages/configuration/users-import/components/atlas-roles.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    <table width="100%" cellspacing="0" data-bind="dataTable:{
+                            data: roles(),
+                            options: {
+                                        dom: 'Bliprt',
```

Comment:
What is this for?

Reply:
It`s a wondering feature of [datatables](https://datatables.net/reference/option/dom),
for example, B means Buttons